ssl/ssl_conf.c

```diff
@@ -600,7 +600,9 @@ static int cmd_DHParameters(SSL_CONF_CTX *cctx, const char *value)
         if (decoderctx == NULL)
             goto end;
         ERR_set_mark();
-        while (!OSSL_DECODER_from_bio(decoderctx, in) && dhpkey == NULL && !BIO_eof(in));
+        while (!OSSL_DECODER_from_bio(decoderctx, in)
+                && dhpkey == NULL
+                && !BIO_eof(in));
```

Comment:
I'm sorry for nitpicking but the indentation is wrong here - either align with ! or add 3 more spaces.